### PR TITLE
feat: header ui with design-system

### PR DESCRIPTION
**Pull request by yjkimjunior**
cc: @goldsteinsveta 

The StatsList component is a bit difficult to use for this case, but as you can see I think the StatItem component works just fine for this header use case.

### CODE REVIEW

**Comment by yjkimjunior** [comment]
pretty insubstantial at the moment so just going to merge to unblock myself.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -20,6 +20,6 @@ linux-test:
     - yarn install --frozen-lockfile
     - yarn build
     - yarn test
-    - yarn lint
+    - yarn lint --fix
```
Comment (amaurymartiny) [inline .gitlab-ci.yml]: I recommend against that, imagine the following scenario:
- Small lint error on your local branch, you commit & push to a PR
- Gitlab run this command, fixes your small lint error on CI
- Gitlab **does not** push this lint fix to your branch or on master
- master has a small lint error

Comment (amaurymartiny) [reply .gitlab-ci.yml]: Would recommend:
- `yarn lint` on CI
- if CI does not pass, run `yarn lint --fix` on your local machine, then push again.
- CI should pass

Comment (yjkimjunior) [reply .gitlab-ci.yml]: ah, i didn't realize the 3rd point would be the case.

**Pull request merged**